serverless/src/submitHarmonyOrder/__tests__/constructOrderPayload.test.js

```diff
@@ -732,8 +732,8 @@ describe('constructOrderPayload', () => {
     })
   })
 
-  describe('when supportsConcatenation = false or enableConcatenateDownload = false', () => {
-    test('constructed payload does not contain concatenate', async () => {
+  describe('when the accessMethod supports concatenation but, it is not selected', () => {
```

Comment:
```suggestion
  describe('when the accessMethod supports concatenation, but it is not selected', () => {
```

Minor grammatical correction